Shared/Flows/Launch/LaunchCoordinator.swift

```diff
@@ -19,9 +19,46 @@ class LaunchCoordinator: PresentableCoordinator<LaunchStatus> {
     override func start() {
         super.start()
 
-        Task {
-            let launchStatus = await LaunchManager.shared.launchApp(with: self.deepLink)
-            self.finishFlow(with: launchStatus)
+        self.startLaunchTask()
+    }
+
+    private var task: Task<Void, Never>?
+
+    private func startLaunchTask() {
```

Comment:
The launch task might have to be run multiple times if there are errors. Separating it out into a function to support that.